common/static/common/js/karma.common.conf.js

```diff
@@ -330,7 +337,11 @@ var getBaseConfig = function (config, useRequireJs) {
         // how many browser should be started simultaneous
         concurrency: Infinity,
 
-        browserNoActivityTimeout: 50000
+        browserNoActivityTimeout: 50000,
+
+        client: {
+            captureConsole: false
```

Comment:
Thanks for the explanation @ehteshamkafeel. It is mildly disconcerting to me that there are errors happening in the browser that will no longer be seen by folks except if they use dev mode (which many developers don't seem to do). I'm wondering how there can be JS stack traces shown in the browser that don't cause a test to fail, and whether they indicate that there is a real issue here. However, you've convinced me that this isn't serious enough, and the alternative of having confusing error messages in the console log seems worse. From the ticket it sounds like @jzoldak is on board with this approach, so I'm fine with it too.